### DISCUSSION TITLE
Naxxramas script

### DISCUSSION THREAD

**Discussion by declart** [chromiecraft/chromiecraft#3490] [Suggestions]
i have found the full scripts of naxxramas that you might use for chromie craft.
it's known that you have to rewrite everything from the beginning to let it work, but i found a mangos vanilla bosses fully scripted, that might be used to have vanilla naxx on the current realm. 
https://github.com/vmangos/core/tree/development/src/scripts/eastern_kingdoms/eastern_plaguelands/naxxramas
check it might be helpful

**Comment by 55Honey** [comment]
Hello declart,
Thanks for your suggestion.
Unfortunately porting a script from one core to another is usually more work than writing it from scratch.

**Comment by declart** [comment]
i don't know much about it, i just thought was a good idea and easier then writing it from zero.

**Comment by Nyeriah** [comment]
Hey,

Thanks for trying to help!

However, the scripts are not an issue, we also have the instance scripted at Azerothcore. The issue is everything else involved in the process of making it completable at level 60. Furthermore, Naxxramas is something that doesn't necessarily align with our plans for progression, as it may trivialize some of the lower TBC content.

Regards.

**Reply by Dupeloop** [reply]
Yes good thing that as a lock in the 64 bracket I will be replacing absolutely non of my aq items. Might want to remove that instance aswell based on ur statement
